### ISSUE TITLE
Cloning an abductor makes it become catatonic.

### ISSUE THREAD

**Issue by fonedsky** [yogstation13/yogstation#2192]
Was an abductor scientist. Agent died. I retrieved my agent and cloned them (because no heart to defib). Once they popped out the cloner, they were catatonic. I died at roundend and the person who was my agent said that they just didn't go back in their body.
Since Abductor species are very different from other ones, it's likely the original coder didn't plan for this scenario and that messed it up.

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
I don't think this is limited to abductors. Happened to me once as a normal lizard.
